src/commands/cmd_json.cc

```diff
@@ -291,11 +291,58 @@ class CommandJsonArrPop : public Commander {
   int64_t index_ = -1;
 };
 
+class CommandJsonArrTrim : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    path_ = args_[2];
+
+    if (args_.size() == 5) {
+      start_ = GET_OR_RET(ParseInt<int64_t>(args_[3], 10));
+      stop_ = GET_OR_RET(ParseInt<int64_t>(args_[4], 10));
+    } else {
+      return {Status::RedisParseErr, errWrongNumOfArguments};
+    }
```

Comment:
I think after https://github.com/apache/kvrocks/pull/1881/files#r1386825383 we don't need this check now.